### DISCUSSION TITLE
Tabbed Full Screen

### DISCUSSION THREAD

**Discussion by TonyValenti** [mRemoteNG/mRemoteNG#2538] [Ideas]
HI mRemoteNG,
It would be really nice if it was possible to have full screen tabs.  I work on a lot of different systems and I like to be full screen but I hate having to minimize full screen to switch PCs.

**Comment by Kvarkas** [comment]
I think its should be posible to add in next version will have a look. I plan to move to .net 8 and they have some nice features regarding such.
